core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -25,6 +25,10 @@
      */
     public static final int DEFAULT_MAX_TTD = 60; // seconds
 
+    /**
+     * The default message size is set to -1, which implies no minimum size is defined.
+     */
+    public static final int DEFAULT_MINIMUM_MESSAGE_SIZE = -1;
```

Comment:
wouldn't 0 (zero) do the same trick but be more *intuitive* in this case?

Reply:
Zero also works. I will update it.